### PR TITLE
Add upgrade guide for 2.4.0

### PR DESCRIPTION
**Pull request by legoktm**
## Status

Work in progress

## Description of Changes

* Bump latest version to 2.4.0
* Copy over instructions for a manual upgrade to Tails 5.0 from 2.3.2
  guide in case people haven't done it yet.
* Announce support for pt_PT with instructions to add it, as well
  as a warning that hi and ro are at risk of being removed.

Refs https://github.com/freedomofpress/securedrop/issues/6440.

## Testing

* Read
* Try out the steps to change languages/locales

## Release 

* Coordinate with 2.4.0 release

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

**Comment by legoktm** [comment]
@cfm I added you as a reviewer for the language support text. The part about pt_PT is overly 🤞🏾 , if it doesn't pan out we'll take it out but have the text saved for 2.5.0.

**Review by cfm — DISMISSED** [review]
Language-related changes look good to me (including `pt_PT` support!).  Thanks for drafting, @legoktm!

**Comment by cfm** [comment]
Appended fc60e611167edd491c3e9e69d935302e04bcddcb from `i18n_tool.py update-docs`.

**Comment by legoktm** [comment]
Does the SecureDrop updater automatically import the new updated signing key? If not we need a manual action item for users to do that.

File: docs/upgrade/2.3.2_to_2.4.0.rst [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  ./securedrop-admin tailsconfig
+
+Upgrade to Tails 5.0
+--------------------
+
+If you have not already done so, you must manually upgrade to Tails 5.0.
+
+.. important::
+
+   You must upgrade your workstations to SecureDrop 2.3.2 or later by following the steps
```
Comment (eloquence) [inline docs/upgrade/2.3.2_to_2.4.0.rst:78]: I would just collapse this as "the latest version of SecureDrop", no reason to keep mentioning 2.3.2.

File: docs/upgrade/2.3.2_to_2.4.0.rst [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+   above *before* upgrading to Tails 5.0. You can verify the version of SecureDrop
+   by running ``git status`` in your ``~/Persistent/securedrop`` directory. The
+   output should include "HEAD detached at 2.3.2" or "HEAD detached at 2.4.0".
```
Comment (eloquence) [inline docs/upgrade/2.3.2_to_2.4.0.rst:81]: As above.

File: docs/upgrade/2.3.2_to_2.4.0.rst (hunk 1 above)
Comment (legoktm) [reply docs/upgrade/2.3.2_to_2.4.0.rst:78]: Done.

**Review by eloquence — APPROVED** [review]
Changes look good to me!

@cfm Thanks for taking care of the screenshots. There are a couple of shots in the Source Guide that are still 2.3.0, I think these are not covered by our functional tests. But IMO can be done later (and maybe we should enumerate those in the RM docs).

**Pull request merged**